src/initialization/InitDistribution.cpp

```diff
@@ -267,6 +209,7 @@ namespace impactx
         refPart.z = 0.0;
         refPart.px = 0.0;
         refPart.py = 0.0;
+        // make the next two lines a helper function?
```

Comment:
@cemitch99 we could make this a helper member function in `RefPart`, what do you think?

Reply:
Discussed on Slack: same for "set_energy_MeV", "get_beta", "get_gamma", etc.?
> Yes, I agree this sounds good.

-> will be a follow-up PR